llarp/config/key_manager.cpp

```diff
@@ -31,10 +31,10 @@ namespace llarp
     if(m_initialized)
       return false;
 
-    m_rcPath           = config.router.ourRcFile();
-    m_idKeyPath        = config.router.identKeyfile();
-    m_encKeyPath       = config.router.encryptionKeyfile();
-    m_transportKeyPath = config.router.transportKeyfile();
+    m_rcPath           = config.router.m_dataDir + "/rc.signed";
```

Comment:
these should use `fs::path` because windows

Reply:
Good point... fixed in db6abd8.